templates/security/certifications.html

```diff
@@ -45,23 +45,23 @@ <h3>Know your defenses</h3>
    </div>
    <div class="col-4 p-divider__block">
      <h3>Access certifications for high security environments</h3>
-     <p>Access to certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and <a href="/cloud/public-cloud">Ubuntu Pro</a> provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
+     <p>Access certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and <a href="/cloud/public-cloud">Ubuntu Pro</a> provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
      <a class="p-button" href="#security-certifications">See our certifications</a>
    </div>
  </div>
 </section>
 
 <section class="p-strip">
-  <div class="row">
+  <div class="row u-vertically-center">
     <div class="col-6">
-      <h2>FIPS certification and CIS compliance with Ubuntu</h2>
-      <p>Learn about Ubuntu CIS and FIPS certified components to enable operating under compliance regimes like FedRAMP, HIPAA, PCI and ISO. Get all of your compliance questions answered in our upcoming webinar to ensure you and your team are, and remain, compliant.</p>
+      <h2>How does Ubuntu enable your compliance with FIPS, and DISA-STIG?</h2>
+      <p>Learn about the US government security standards and the common challenges faced by organiszations in their implementation. See how the <a href="/security/certifications/docs/disa-stig">Ubuntu Security Guide</a> can transform systems compliance in a few minutes. Get to know how Ubuntu is a secure platform for government agencies and complying organiszations to build, operate and innovate with open source applications and technologies.</p>
```

Comment:
These were in the copy doc too, I've updated them there:
```suggestion
      <p>Learn about the US government security standards and the common challenges faced by organisations in their implementation. See how the <a href="/security/certifications/docs/disa-stig">Ubuntu Security Guide</a> can transform systems compliance in a few minutes. Get to know how Ubuntu is a secure platform for government agencies and complying organisations to build, operate and innovate with open source applications and technologies.</p>
```